jiant/tasks/tasks.py

```diff
@@ -3731,3 +3732,121 @@ def get_metrics(self, reset=False):
+            "test": os.path.join(path, "test.txt"),
+        }
+
+    def get_data_iter(self, path):
+        """Loading data file and tokenizing the text
+        Args:
+            path: (str) data file path
+        """
+        count = 0
+        with open(path, encoding="utf-8") as txt_fh:
+
+            for row in txt_fh:
+                count += 1
+                if count == 10:
+                    break
+                toks = row.strip()
+                sentences = row.split(".")
+                if len(sentences) <= 1:
+                    continue
+                else:
+                    for i in range(len(sentences) - 1):
+                        if random.uniform(0, 1) > 0.5:
+                            is_right_order = 1
+                            sent_a = sentences[i]
+                            sent_b = sentences[i + 1]
+                        else:
+                            is_right_order = 0
+                            sent_a = sentences[i + 1]
+                            sent_b = sentences[i]
+                        sent_a_processed = tokenize_and_truncate(
+                            self._tokenizer_name, sent_a, self.max_seq_len // 2
+                        )
+                        sent_b_processed = tokenize_and_truncate(
+                            self._tokenizer_name, sent_b, self.max_seq_len // 2
+                        )
+                        yield (sent_a_processed, sent_b_processed, is_right_order)
+
+    def load_data(self):
+        pass
+
+    def process_split(
+        self, split, indexers, model_preprocessing_interface
+    ) -> Iterable[Type[Instance]]:
+        """Process a sentence order prediction split by indexing and creating fields.
+        Args:
+            split: (list) a single list of sentences
+            indexers: (Indexer object) indexer to index input words
+        """
+
+        def _make_instance(sent_):
+            sent_a, sent_b, is_right_order = sent_
+            if model_preprocessing_interface.model_flags["uses_pair_embedding"]:
+                inp = model_preprocessing_interface.boundary_token_fn(sent_a, sent_b)
+                input_sent = sentence_to_text_field(inp, indexers)
+                label = LabelField(is_right_order, label_namespace="labels", skip_indexing=True)
+                d = {"inputs": input_sent, "labels": label}
+            else:
+                inp1 = sentence_to_text_field(
+                    model_preprocessing_interface.boundary_token_fn(sent_a), indexers
+                )
+                inp2 = sentence_to_text_field(
+                    model_preprocessing_interface.boundary_token_fn(sent_b), indexers
+                )
+                label = LabelField(is_right_order, label_namespace="labels", skip_indexing=True)
+                d = {"input1": inp1, "input2": inp2, "targs": label}
+            return Instance(d)
+
+        for sent in split:
+            yield _make_instance(sent)
+
+    def count_examples(self):
```

Comment:
Same here—not sure what the history/intent is behind when this is/isn't overridden.